docs/src/pages/components/data-grid/style/StylingAllCells.tsx

```diff
@@ -37,7 +41,7 @@ export default function StylingAllCells() {
       <DataGrid
         rows={rows}
         columns={columns}
-        getCellClassName={(params: GridCellParams) => {
+        getCellClassName={(params: GridCellOptionsParams) => {
```

Comment:
Do we want to explicitly set the `params` type here ?
TypeScript can infer it from the `getCellClassName` signature.

Reply:
It's docs, since this interface is exposed people can also see what is inside it.